### PR TITLE
[www] make 404 great again

### PR DESCRIPTION
**Pull request by fabi1cazenave**
Looks like I’ve degraded our beloved 404 page with #142. Here’s a fix.

### CODE REVIEW

File: www/assets/css/theme.css [hunk 1]
```diff
@@ -92,6 +92,10 @@ body {
 }
 
 main {
+  display: block; /* using 'flex' here would mess with `margin: auto` blocks */
```
Comment (fabi1cazenave) [inline www/assets/css/theme.css:95]: I’d appreciate some help here.

Setting `display: flex` on `<main>` would be neat to balance the main content in short pages (e.g. 404, Duck Typist…) but it messes with block elements carrying a `margin: [whatever] auto` property (e.g. `<figure>` elements) and I don’t understand why.

Does anyone have a clue here ?

Comment (cedricr) [reply www/assets/css/theme.css:95]: Not a direct answer to this particular problem, but it looks to me that all that is to compensate the fact that the main `nav` is not inside the `header` tag, and so their flex container (`body`) will separate them if you don’t force `main` to take all the space.
So, just put the nav inside the header, no?

Comment (cedricr) [reply www/assets/css/theme.css:95]: oh, that’s because of the sticky…

Comment (cedricr) [reply www/assets/css/theme.css:95]: can’t you put the sticky on the whole header, but not make it start at 0, so that only the nav will stick ?

Comment (fabi1cazenave) [reply www/assets/css/theme.css]: Nope, that won’t work either. Tried it already. :-)

Comment (cedricr) [reply www/assets/css/theme.css]: otherwise you can add a "width: 100%" on your figures...

**Pull request merged**